contracts/modules/LoanClosingsShared.sol

```diff
@@ -392,29 +608,79 @@ contract LoanClosingsShared is
         } else {
             /// loanCloseAmount is calculated after swap; for this case we want to swap the entire source amount
             /// and determine the loanCloseAmount and withdraw amount based on that.
+            loanCloseAmount = 0;
             loanCloseAmountLessInterest = 0;
         }
```

Comment:
no need to explicitly assign loanCloseAmount = 0 here, as Solidity will default-initialize it to zero.
the variable is declared in the return values and not used before assignment in this branch.  
also added `return()` for clarity
```suggestion
            loanCloseAmountLessInterest = 0;
        }
        return(loanCloseAmount, loanCloseAmountLessInterest);
```

Reply:
Updated the comment [here](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/559/commits/b443f173e56acaf86d501b7f5f8249c14bc739b7)